controllers/virtualmachine/volumebatch/utils/constants.go

```diff
@@ -0,0 +1,11 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
```

Comment:
Can you move these out of the batch directory since this is shared by both version of controllers?